### PR TITLE
sync master [0.7.2] 

### PR DESCRIPTION
**Pull request by ca333**
Komodo 0.7.2 [Longpan]

### CODE REVIEW

**Comment by dimxy** [comment]
I went over the code in this PR and did not find any issues (apart from that small issue with libcc mentioned above).
As there is some notarisation-related code refactored in this PR I'd suggest that some existing NN operator would run his server for some time on this code to test it

**Review by DeckerSU — APPROVED** [review]
As all changes have been verified on the accepting to the `dev` branch stage, or within `hfnet` testing, and passing through `dev->beta->master` haven't new changes -> LGTM.

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -2208,8 +2093,18 @@ bool CWallet::IsMine(const CTransaction& tx)
     return false;
 }
 
+/*
+    Before Verus changes CWallet::IsMine(const CTransaction& tx) called other version
+    of IsMine for each vout: CWallet::IsMine(const CTxOut& txout), so now we have two
+    similar functions:
+        - isminetype CWallet::IsMine(const CTransaction& tx, uint32_t voutNum) (wallet.cpp)
+        - isminetype IsMineInner(const CKeyStore& keystore, const CScript& _scriptPubKey, IsMineSigVersion sigversion) (wallet_ismine.cpp)
+    TODO: sort this out (!)
```
Comment (Alrighttt) [inline src/wallet/wallet.cpp:2102]: Did we "sort this out"?

Comment (DeckerSU) [reply src/wallet/wallet.cpp:2102]: No, we postponed it.

**Review by tonymorony — APPROVED** [review]

**Pull request merged**